frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -715,68 +717,76 @@ const ManagePolicyPage = ({
   };
 
   const renderMainTable = () => {
-    return !isRouteOk || (isPremiumTier && !userTeams) ? (
-      <Spinner />
-    ) : (
+    if (!isRouteOk || (isPremiumTier && !userTeams)) {
+      return <Spinner />;
+    }
+    if (isAllTeamsSelected) {
```

Comment:
Changing this switch from using `isAnyTeamSelected` to `isAllTeamsSelected` is the only logical change  in `renderMainTable`, the rest is reorganization.